src/pages/location.tsx

```diff
@@ -0,0 +1,36 @@
+import Image from 'next/image';
+
+import { Meta } from '../layout/Meta';
+import styles from '../styles/location.module.css';
+import { Main } from '../templates/Main';
+
+const Location = () => (
+  <Main meta={<Meta title="ITEZ" description="ITEZ Frontend" />}>
+    <div className=" h-screen">
+      {' '}
+      <div className="flex  flex-col  w-full h-full ">
+        <div className={styles.grad}>
+          <div className="">
+            <h1>Location</h1>
+            <div className="flex ">
+              <Image
+                className=" justify-center z-10 "
+                src="/map.svg"
```

Comment:
we can not have a map as an svg @cazterk 
you might want to look through some examples 
https://cheatcode.co/tutorials/how-to-render-a-map-with-markers-using-google-maps-in-next-js

Reply:
that was just a placeholder to test out the layout and the ticket for the location is on prince's side